MapPointMapper/ViewController.swift

```diff
@@ -43,8 +44,8 @@ class ViewController: NSViewController, MKMapViewDelegate, NSTextFieldDelegate {
       self.readFileAtURL(openPanel.URL)
     })
   }
-  
-  @IBAction func addLineFromTextPressed(sender: NSButton) {
+
+  @IBAction func addLineFromTextPressed(sender: NSObject) {
```

Comment:
I bound both the text field and nsbutton to this as `Sent Action` and `Reference Outlet`.  Since They are triggered by potentially different objects, `NSObject` should cover both senders